scrypto-unit/src/test_runner.rs

```diff
@@ -477,11 +477,20 @@ impl<E: NativeVmExtension, D: TestDatabase> Drop for TestRunner<E, D> {
             .check_db(&self.database)
             .expect("Database should be consistent");
 
+        // Defining a composite checker of all of the application db checkers we have.
+        radix_engine::define_composite_checker! {
```

Comment:
Would be better I think if the name of the composite checker was one of the arguments into the macro.

Reply:
Sounds good, making that change.